src/app/admin/admin-registries/metadata-schema/metadata-field-form/metadata-field-form.component.ts

```diff
@@ -98,25 +98,35 @@ export class MetadataFieldFormComponent implements OnInit, OnDestroy {
    * Initialize the component, setting up the necessary Models for the dynamic form
    */
   ngOnInit() {
-    combineLatest(
+    combineLatest([
       this.translateService.get(`${this.messagePrefix}.element`),
       this.translateService.get(`${this.messagePrefix}.qualifier`),
       this.translateService.get(`${this.messagePrefix}.scopenote`)
-    ).subscribe(([element, qualifier, scopenote]) => {
+    ]).subscribe(([element, qualifier, scopenote]) => {
       this.element = new DynamicInputModel({
         id: 'element',
         label: element,
         name: 'element',
         validators: {
           required: null,
+          pattern: '^[^.]*$',
```

Comment:
I think we should block commas and spaces here as well. (But should allow underscores)  So, this should be:  `'^[^. ,]*$',`

I tested and was able to add a metadata field with a space in the element and qualifier. That seems wrong.